config/wallet/profileImporter.html

```diff
@@ -0,0 +1,60 @@
+<!--
+  ~ Copyright 2020 NEM Group (https://nem.io)
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and limitations under the License.
+  ~
+  -->
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+  <title>Profile Importer</title>
+  <script type="application/javascript">
+    window.onload = function () {
+      //save entered gmail address
+      document.getElementById('save').addEventListener(
+        'click',
+        function () {
+          try {
+            const profile = JSON.parse(document.getElementById('profile').value)
+            localStorage.setItem('profiles', JSON.stringify(profile.profiles))
+            localStorage.setItem('accounts', JSON.stringify(profile.accounts))
```

Comment:
Could you also save (and restore) the `mosaicConfiguration` setting?
It stores whether each mosaic is shown in each account's assets list, and it's helpful when we're not interested in some of the mosaics.

Reply:
Added